src/packet.cpp

```diff
@@ -103,7 +111,7 @@ const std::vector<uint8_t> Packet::makeSendBuffer(const std::string& secret)
     return sendBuffer;
 }
 
-Attribute* Packet::makeAttribute(uint8_t type, const uint8_t* data, size_t size, const std::string& secret, const std::array<uint8_t, 16>& auth)
+RadProto::Attribute* Packet::makeAttribute(uint8_t type, const uint8_t* data, size_t size, const std::string& secret, const std::array<uint8_t, 16>& auth)
```

Comment:
This function does not use any class members, so it should be either a free function, or a static member. It is only used in `Packet` constructor, so, ideally, it should be a local function in the anonymous namespace.